invokeai/frontend/web/src/features/gallery/components/ImageGrid/GalleryImage.tsx

```diff
@@ -145,6 +155,19 @@ const GalleryImage = (props: HoverableImageProps) => {
           onMouseOut={handleMouseOut}
         >
           <>
+            {isHovered && (
+              <Text
+                position="absolute"
+                background="black"
+                color="white"
+                fontSize="12px"
+                bottom={1.5}
+                right={8}
+                px={2}
+                opacity={0.7}
+                borderRadius="base"
```

Comment:
Please use the theme tokens for colors and font sizes.

Also - because there is already an icon @ bottom-right, this should be at the bottom left

Reply:
I added the space between both icons present at bottom right, will that not work?

Reply:
Fixed with latest commit!